testing/test_compat.py

```diff
@@ -91,9 +90,6 @@ def test_is_generator_asyncio():
     result.stdout.fnmatch_lines(["*1 passed*"])
 
 
-@pytest.mark.skipif(
-    sys.version_info < (3, 5), reason="async syntax available in Python 3.5+"
-)
 def test_is_generator_async_syntax(testdir):
```

Comment:
I wonder what `is_generator` should return for an async gen function...

Reply:
It should return False - because it's used to check for yield fixtures. I'll add a test